docs/UserGuide.md

```diff
@@ -112,14 +113,14 @@ Format: `edit INDEX [n/NAME] [i/IC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/REMARK] [
   specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 i/S1111112B e/johndoe@example.com` Edits the phone number,IC, email address of the 1st person to be `91234567`, ` i/S1111112B` and johndoe@example.com` respectively.
+*  `edit 1 p/91234567 i/S1111112B e/johndoe@example.com` Edits the phone number,NRIC, email address of the 1st person to be `91234567`, ` i/S1111112B` and johndoe@example.com` respectively.
```

Comment:
Perhaps S1111112B could be replaced with S1234567B so that it's easier to check the correct format?